playbooks/roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# ==============================================================================
+# Constants
+# ==============================================================================
+# 360 * 10 seconds = 1hour
+wait_for_zowe_service_retries: 60
+# Every 10 seconds
+wait_for_zowe_service_delay: 5
+# zowe docker image
+zowe_docker_image: ompzowe/zowe-v1-lts
+zowe_docker_tag: amd64
+
+# ==============================================================================
+# Variables should be verified and overwrittern.
+# ==============================================================================
+# default z/OSMF port. Optional, default value is ansible_ssh_host
+zos_zosmf_host:
+# default z/OSMF port
+zos_zosmf_port: 10443
+zowe_zss_https: False
+zowe_zss_port: 8542
+zowe_launch_components: zss
+zowe_sanity_test_testcases: '-- --config .mocharc-docker.yml'
```

Comment:
I believe this value doesn't need to be here since the verify playbook/role won't read this config file.

Reply:
removed